src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/utils/Constants.java

```diff
@@ -17,7 +17,7 @@ public class Constants {
     public static final String TRANSACTIONS_PRIVATE_API_URI_PREFIX = "/private/transactions/";
     public static final String VALIDATION_STATUS_URI_SUFFIX = "/validation-status";
     public static final String URL_GET_PARTNERSHIP = "/transactions/%s/limited-partnership/partnership/%s";
-    public static final String URL_RESUME_PARTNERSHIP = "/limited-partnerships/registration/transaction/%s/submission/%s/which-type";
+    public static final String URL_RESUME_PARTNERSHIP = "/limited-partnerships/transaction/%s/submission/%s/resume";
```

Comment:
This URL is really being used to resume a registration not a partnership. Could change the constant name to reflect this but equally could be done later, when resuming other journeys - transition etc.